.hooks/README.md

```diff
@@ -0,0 +1,17 @@
+# Git Hooks
+
+Run conventional commit checks locally without any tool installation
+
+## Usage
+
+Execute the below command in the root directory of the project:
+
+```sh
+ln -s -f ../../.hooks/commit-msg ./.git/hooks/commit-msg
+```
+
+Make the file executable:
+
+```sh
+chmod +x ./.hooks/commit-msg
+```
```

Comment:
I just tested this locally and didn't have to make the file executable since it is already checked in with the executable bit set.

```suggestion
```

Reply:
I will edit docu as `troubleshooting` for executable :)